Eatery/Views/Eateries/SearchResultsTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SearchResultsTableViewCell.swift
+//  Eatery
+//
+//  Created by William Ma on 4/22/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class SearchResultsTableViewCell: UITableViewCell {
+
+    private(set) var favoriteButton: UIButton!
```

Comment:
You can say `private let favoriteButton = UIButton(type: .system)` to be more concise.